### PR TITLE
update about section for doc/dev url

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,8 +45,16 @@ about:
...
   license_file: COPYING
-  summary: An extensible I/O abstraction library for dealing with structured file formats
+  license_url: https://github.com/GNOME/libgsf/blob/master/COPYING
+  summary: The G Structured File Library
+  description: |
+    The project aims to provide an efficient extensible i/o abstraction
+    for dealing with different structured file formats.
+  doc_url: https://github.com/GNOME/libgsf
+  doc_source_url: https://github.com/GNOME/libgsf/tree/master/doc
+  dev_url: https://github.com/GNOME/libgsf
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: The correct home, doc_url, and dev_url are `https://gitlab.gnome.org/GNOME/libgsf`. The github link is a mirror

File: recipe/meta.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ package:
   version: {{ version }}
 
 source:
-  url: https://ftp.gnome.org/pub/GNOME/sources/{{ name }}/{{ ".".join(version.split(".")[:2]) }}/{{ name }}-{{ version }}.tar.xz
+  url: https://download.gnome.org/sources/{{ name }}/{{ ".".join(version.split(".")[:2]) }}/{{ name }}-{{ version }}.tar.xz
   sha256: 6e6c20d0778339069d583c0d63759d297e817ea10d0d897ebbe965f16e2e8e52
 
 build:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:12]: Pinning for requirements is old enough (README.md is modified 4 years ago):
```
    glib	>= 1.3.10
    zlib	>= 1.1.3
    libxml2	>= 2.4.16
```
https://gitlab.gnome.org/GNOME/libgsf/-/blob/LIBGSF_1_14_50/README

Comment (skupr-anaconda) [reply recipe/meta.yaml:12]: Do we need `gobject-introspection` as a dependency? See https://gitlab.gnome.org/GNOME/libgsf/-/blob/LIBGSF_1_14_50/configure.ac#L74

File: recipe/meta.yaml (hunk 1 above)
Comment (katietz) [reply recipe/meta.yaml]: hmm, to access the gitlib.gnome.org page you need to a login for gitlab

Comment (katietz) [reply recipe/meta.yaml]: thanks for pointing our that I had here actually doc_source_url, and license_url tags in recipe.  I just removed them

File: recipe/meta.yaml (hunk 2 above)
Comment (katietz) [reply recipe/meta.yaml:12]: I think no.  gobject-introspecition is ok to be skipped.
  The other pinnings seems to be not necessary to me as using the newest variants of glib/zlib/libxml2 is fine.  I don't see a good reason to pin here, as we have no package specifying an upperbound on those packages.

**Review by danpetry — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**